addons/lunch/static/tests/lunch_kanban_tests.js

```diff
@@ -18,11 +18,9 @@ async function makeLunchView(extraArgs = {}) {
             arch: `
             <kanban js_class="lunch_kanban">
                 <templates>
-                    <t t-name="kanban-box">
-                        <div>
-                            <field name="name"/>
-                            <field name="price"/>
-                        </div>
+                    <t t-name="kanban-card" class="flex-row">
```

Comment:
you can certainly remove the flex-row here (in tests, we want minimal, as simple as possible, archs)